modules/localgov_expired_events/localgov_expired_events.module

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use Drupal\content_moderation\Entity\ContentModerationState;
+use \Drupal\node\Entity\Node;
+/**
+ * @file
+ * Contains localgove_expired_events.module.
+ */
+function localgov_expired_events_cron() {
+  $config = \Drupal::config('localgov_expired_events.settings');
+  $expire_days = $config->get('expire_days');
+  $items_per_cron = $config->get('items_per_cron');
+  $action = $config->get('action');
+  $limit = '';
+
+  if ($items_per_cron > 0 ) {
+    $limit = " LIMIT {$items_per_cron}" ;
+  }
+  if ($action == "unpublished") {
+    $node_is_published = "  AND node_field_data.status = 1";
+  }
+
+  try {
+      $statement = "SELECT * FROM date_recur__node__localgov_event_date 
+                    LEFT JOIN node_field_data
+                    ON node_field_data.nid = date_recur__node__localgov_event_date.entity_id
+                    WHERE DATE_ADD(localgov_event_date_end_value, INTERVAL {$expire_days} DAY ) < NOW()" . $node_is_published . $limit;
+      $database = \Drupal::database();
+      $query = $database->query($statement);
```

Comment:
It's best practice to use Drupal's query builder, rather than join strings together for a dynamic query like this. Is it possible to use an entity query for it though? (It may not, on account of how date_recur works). That would be better, as it means the table names don't get included in this module's code, just the field names.